src/dispatch/signal/flows.py

```diff
@@ -291,6 +291,9 @@ def _should_update_signal_message(signal_instance: SignalInstance) -> bool:
     """
     global _last_nonupdated_signal_cache
 
+    if signal_instance.case_id:
```

Comment:
Separately, this function should run for cases with a `case_id`, since it is responsible for updating the `Total instances in this case: X` text in the Case Slack thread. This will cause it to never update.

Reply:
Done.